.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Package
+
+on:
+  push:
+    branches:
+      - develop
+
+env:
```

Comment:
Are these still required even if we are not building STK?

Reply:
`ADAM_STK_LICENSE_DATA`, I think, is used as part of the unit tests. The license utility looks in the environment variables for the license contents. You can try removing the env variable here and see if the GH actions pass.

The `SERVER_PROPERTIES_PATH` doesn't seem to be used anywhere, so you can remove it.

Reply:
ADAM_STK_LICENSE_DATA is required. SERVER_PROPERTIES_PATH is not required.